3rdparty/udis86/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# UJIT: CMake file for udis86 to avoid dependencies on autoconf and libtool.
+# UJIT: Based on https://github.com/vmt/udis86/pull/99/files by hasherezade.
+# UJIT: Copyright (C) 2020 Anton Soldatov.
+# UJIT: Only the library is built, CLI is irrelevant for this project.
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+project(udis86 C)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -Werror -Wall -Wextra -pedantic")
```

Comment:
Why is it necessary to explicitly add flags there? CMAKE_C_FLAGS might already have all these flags set and the user might want to compile everything, including deps as c17, for example.

Reply:
Will be fixed.